applications/asset_tracker_v2/src/modules/Kconfig.cloud_module

```
@@ -20,6 +20,16 @@ config NRF_CLOUD_AGPS
 	depends on GNSS_MODULE
 	default y
 
+if NRF_CLOUD_PGPS && !NRF_CLOUD_PGPS_PARTITION
+
+config CLOUD_PGPS_MCUBOOT_PARTITION
+	bool "Use mcuboot secondary partition for P-GPS data"
```

This isn't supposed to be configured by the user, so maybe we could remove the prompt?
```suggestion
	bool
```

---

Good point.